podcast/2.15.md

```diff
@@ -0,0 +1,1299 @@
+
+**Sugu Sougoumarane:**
+
+For the longest time, I never intended to start a company, because why?
+
+**Beyang Liu:**
+
+Yeah. Don't start a company for the sake of starting a company.
+
+**Sugu Sougoumarane:**
+
+For the sake of starting a company. I was actually previously approached by VCs, who said, "Oh my God, what are you sitting on? You should start a company." I said, "No, not interested." But I guess what the VCs saw–they saw the writing on the wall. They were trying to even convince me and I wasn't sold, but I guess I also started seeing the writing on the wall.
+
+**Beyang Liu:**
+
+Which VCs? Do you feel comfortable sharing them? Were there any really good ones in the beginning?
+
+**Sugu Sougoumarane:**
+
+Yeah. Actually, Amplify Partners was one of them.
+
+**Beyang Liu:**
+
+Oh, nice. Amplify is awesome. We work with them as well.
+
+**Sugu Sougoumarane:**
+
+Our initial round was not by them, but they did plant the idea. And we did talk to them. We told them "You told us three years ago and here I am."
+
+**Beyang Liu:**
+
+Yeah. Okay. So that got you kind of thinking about the...
+
+**Sugu Sougoumarane:**
+
+Kind of. It was in the back of my mind.
+
+I was even willing to encourage other people to start a company and say, "If you start a company, I'll help you." From within YouTube. I don't care.
+
+**Beyang Liu:**
+
+Wow.
+
+**Sugu Sougoumarane:**
+
+But nobody did. We felt that for YouTube to continue to use Vitess, it had to become a mainstream project in its own right. So that was one motivation.
+
+**Beyang Liu:**
+
+Yeah. Keep it around.
+
+**Sugu Sougoumarane:**
+
+Yeah. But then there were other motivations. The other motivation was eventually what is happening is, YouTube was also changing as a company, where the newer and newer talent that was coming in, they were all product-oriented.
+
+Where does YouTube make money? YouTube does not make money by building Vitess.
+
+YouTube makes money by producing videos, publishing them, and stuff. That's where, naturally, all the effort was going. And at some point in time, the newer management in YouTube was starting to ask, why are we funding this project? But then this project is gaining adoption outside. Everybody is using it. We need this to be funded.
+
+And then there is Spanner, and Google internally was saying, "We don't know what this Vitess is."
+
+**Beyang Liu:**
+
+Just put it all in Spanner.
+
+**Sugu Sougoumarane:**
+
+Put it all in Spanner. That is what is blessed within Google. You should move to that. So I started talking to people. I talked to Joe Beda and Craig McLuckie, who were the founders of Heptio. So I started talking to him, because they kind of went through the same thing. They created Kubernetes, and ended up leaving YouTube to start Heptio as a company. And they also founded the CNCF.
```

Comment:
```suggestion
Put it all in Spanner. That is what is blessed within Google. You should move to that. So I started talking to people. I talked to Joe Beda and Craig McLuckie, who were the founders of Heptio. So I started talking to them, because they kind of went through the same thing. They created Kubernetes, and ended up leaving YouTube to start Heptio as a company. And they also founded the CNCF.
```